### PR TITLE
webdriverio: fix control key w3c compliancy

### PR DESCRIPTION
**Pull request by LukoyanovE**
## Proposed changes

[//]: # (Describe the big picture of your changes here to communicate to the maintainers why we should accept this pull request. If it fixes a bug or resolves a feature request, be sure to link to that issue.)

Cannot send [w3c described](https://w3c.github.io/webdriver/#keyboard-actions) `Control` key.
Somewhy it's called `'Control Left'` in WDIO.

Also added `";", "=", "0", "1", "2", "3", "4", "5", "6", "7", "8", "9", "*", "+", ",", "-", ".", "/",` key codes.

There are some additional "right" keys in w3c spec, but I don't know how to add it to WDIO:
```
\uE050	"Shift"
\uE051	"Control"
\uE052	"Alt"
\uE053	"Meta"
\uE054	"PageUp"
\uE055	"PageDown"
\uE056	"End"
\uE057	"Home"
\uE058	"ArrowLeft"
\uE059	"ArrowUp"
\uE05A	"ArrowRight"
\uE05B	"ArrowDown"
\uE05C	"Insert"
\uE05D	"Delete"
```

## Types of changes

[//]: # (What types of changes does your code introduce to WebdriverIO?)
[//]: # (_Put an `x` in the boxes that apply_)

- [x] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist

[//]: # (_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._)

- [x] I have read the [CONTRIBUTING](https://github.com/webdriverio/webdriverio/blob/master/CONTRIBUTING.md) doc
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

## Further comments

[//]: # (If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...)

### CODE REVIEW

File: packages/webdriverio/src/constants.js [hunk 1]
```diff
@@ -327,23 +328,41 @@ export const UNICODE_CHARACTERS = {
     'ArrowDown': '\uE015',
     'Insert': '\uE016',
     'Delete': '\uE017',
+    ';': '\uE018',
```
Comment (christian-bromann) [inline packages/webdriverio/src/constants.js]: Why do we need to define these twice?

Comment (LukoyanovE) [reply packages/webdriverio/src/constants.js]: [there](https://webdriver.io/docs/api/browser/keys.html) is doc says:

> You’ll find all supported characters [here](https://w3c.github.io/webdriver/webdriver-spec.html#keyboard-actions). 

And there are these codes: `";", "=", "0", "1", "2", "3", "4", "5", "6", "7", "8", "9", "*", "+", ",", "-", ".", "/",`.

I'm looking into `grapheme-splitter` and it it just split all and send such symbols as is.

Ok maybe, I should delete such symbols from PR.
I'll do, if you say.

Comment (christian-bromann) [reply packages/webdriverio/src/constants.js]: yes but these are all defined twice .. if you want to use these special characters you e.g. use `Semicolon` or just send `;` as a string.

Comment (LukoyanovE) [reply packages/webdriverio/src/constants.js]: Maybe doc should be rewritten to other key list, instead of w3c one?

**Comment by CrispusDH** [comment]
Might be better to put at least one test for check random control key?

File: packages/webdriverio/src/constants.js (hunk 1 above)
Comment (christian-bromann) [reply packages/webdriverio/src/constants.js]: Yes, we should link to the constant table. I honestly don't know what the difference between sending `Numpad 0` and `0`

Comment (LukoyanovE) [reply packages/webdriverio/src/constants.js]: `Numpad<N>` should be a keycode for separate Numpad part of keyboard

W3C spec includes it too
https://w3c.github.io/webdriver/#keyboard-actions

It should return the same code as `<N>` key, but when shifted, the same as `Insert`

**Comment by LukoyanovE** [comment]
@CrispusDH , random control key?
it's question about all keys, or some special, like `Control`, `Right Control`, `Alt`, and others?

**Comment by CrispusDH** [comment]
As I understand, currently, we have zero tests for that. So, at least one that will check each time random item from `UNICODE_CHARACTERS` it would be great :). @christian-bromann what do you think?

**Pull request merged**